app/payloads/task_token.go

```diff
@@ -25,7 +25,7 @@ type TaskToken struct {
 	HintPossible       *bool              `json:"bHintPossible,omitempty"`
 	HintsRequested     *string            `json:"sHintsRequested,omitempty"`
 	HintsGivenCount    *string            `json:"nbHintsGiven,omitempty"`
-	AccessSolutions    *string            `json:"bAccessSolutions,omitempty"` // "0" or "1"
+	AccessSolutions    *formdata.Anything `json:"bAccessSolutions,omitempty"` // "0" or "1" / 0 or 1
```

Comment:
It can, but I'm not sure we want to validate it always come from AlgoreaBackend. The fields like this one will be reworked very soon, so that they become '*bool' or even 'bool'

Reply:
👍